activities.js

```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
- /** Version: 0.2.0 */
+ /** Version: 0.3.0 */
```

Comment:
We will have to make this either 0.40.0 or 0.3.1 since NPM deployed version is already 0.3.0.

Reply:
Done